.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: Go
+on: [push]
+jobs:
+  run:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    env:
+      OS: ${{ matrix.os }}
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        id: go
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v1
+
+      - name: Get dependencies
+        run: |
+          go get -v -t -d ./...
+
+      - name: Build compliance-masonry
+        run: go build cmd/compliance-masonry/compliance-masonry.go
+
+      - name: Build masonry
+        run: go build cmd/masonry/masonry.go
+
+      - name: Run tests
+        run: |
+          go test $(go list ./...) -covermode=count -coverprofile=coverage.txt
+          go tool cover -func coverage.txt
+
+      - name: Upload coverage to Codecov
```

Comment:
again, purely stylistic, feel free to ignore/throw it back, but [for consideration, we should be able to get away with removing the `name` key entirely](https://github.com/codecov/codecov-action#usage) and having github actions use the default identifier of the step-- but by the same token, the same codecov-action example page has it included, so it's [really just style, no worries either way](https://github.com/codecov/codecov-action#example-workflowyml-with-codecov-action)

Reply:
As previously mentioned, prefer to keep it for readability.